### PR TITLE
Add PMTCT opt-out app

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
- add flow
- add tests

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
all commits to this points came from a merge with open issue 1 (setup base)

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps @gsvr plear review

File: src/ussd_pmtct_optout.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        self.init = function() {
+            // init services
+        };
+
+    // TEXT CONTENT
+        var questions = {
+            "state_timed_out":
+                $("You have an incomplete registration. Would you like to continue with this registration?"),
+            "state_optout_reason_menu":
+                $("Why do you no longer want to receive messages related to keeping your baby HIV-negative:"),
```
Comment (gsvr) [inline src/ussd_pmtct_optout.js]: replace `:` with `?`

File: src/ussd_pmtct_optout.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                characters_per_page: 182,
+                options_per_page: null,
+                more: $('More'),
+                back: $('Back'),
+                // error: ,
+                choices: [
+                    new Choice("not_hiv-pos", $("I am not HIV-postive")),
+                    new Choice("miscarriage", $("I had a miscarriage")),
+                    new Choice("stillborn", $("My baby was stillborn")),
+                    new Choice("passed_away", $("My baby passed away")),
```
Comment (gsvr) [inline src/ussd_pmtct_optout.js]: Please use the same values for these choices as we did in MomConnect.

File: src/ussd_pmtct_optout.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        self.add("state_optout_reason_menu", function(name) {
+            return new PaginatedChoiceState(name, {
+                question: questions[name],
+                characters_per_page: 182,
+                options_per_page: null,
+                more: $('More'),
+                back: $('Back'),
+                // error: ,
+                choices: [
+                    new Choice("not_hiv-pos", $("I am not HIV-postive")),
```
Comment (gsvr) [inline src/ussd_pmtct_optout.js]: avoid the dash?

**Comment by gsvr** [comment]
@NicolaasDieBaas Small comments then :+1:

File: go-app-ussd_pmtct_optout.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+        App.call(self, "state_start");
+        var $ = self.$;
+        // var interrupt = true;
+
+        self.init = function() {
+            // init services
+        };
+
+    // TEXT CONTENT
+        var questions = {
```
Comment (gsvr) [inline go-app-ussd_pmtct_optout.js]: I'd prefer just having the content in the states.

**Pull request merged**